ganttproject/src/net/sourceforge/ganttproject/GPTreeTableBase.java

```diff
@@ -346,13 +345,22 @@ protected ColumnImpl findColumnByViewIndex(int index) {
     private final JXTreeTable myTable;
     private final TableColumnExt myTableColumn;
     private final Column myStub;
+    private int sort;
 
     protected ColumnImpl(JXTreeTable table, TableColumnExt tableColumn, ColumnList.Column stub) {
       myTable = table;
       myTableColumn = tableColumn;
       myStub = stub;
     }
 
+    public int getSort() {
+      return sort;
```

Comment:
please follow [our style guide](https://docs.ganttproject.biz/development/#_change_process): `mySort`. I also recommend using either boolean or (better) enum for sorting. `getSort() == SortOrder.ASC` is way more readable than `getSort() == 1`